### PR TITLE
Manual sync as separate screen.

### LINKED ISSUE #2692: Demo app ui changes.
**Is your feature request related to a problem? Please describe.**
1. Manual sync as separate screen.
2. Remove Navigation drawer.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Would you like to work on the issue?**
Please state if this issue should be assigned to you or who you think could help to solve this issue.

### PR DESCRIPTION
**Pull request by santosh-pingle**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #2692 

**Description**

- Manual sync as separate screen.
- Remove navigation drawer.

**Alternative(s) considered**
Have you considered any alternatives? And if so, why have you chosen the approach in this PR?

**Type**
Choose one: (Bug fix | Feature | Documentation | Testing | Code health | Builds | Releases | Other)

**Screenshots (if applicable)**

**Checklist**

- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing](https://google.github.io/android-fhir/contrib/) page.
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate ).
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach.
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project.
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally.
- [x] I have built and run the demo app(s) to verify my change fixes the issue and/or does not break the demo app(s).

### CODE REVIEW

**Comment by santosh-pingle** [comment]
Uploading Screen Recording 2024-10-09 at 3.19.27 PM.mov…

File: demo/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -68,4 +68,7 @@
   <string name="last_sync">Last sync</string>
   <string name="syncing">Syncing</string>
   <string name="sync">Sync</string>
+  <string name="last_sync_timestamp">Last sync: %1$s</string>
+  <string name="current_status">Sync status: %1$s</string>
+  <string name="manual_sync">Manual Sync</string>
```
Comment (jingtang10) [inline demo/src/main/res/values/strings.xml:73]:
```suggestion
  <string name="manual_sync">Sync</string>
```

File: demo/src/main/java/com/google/android/fhir/demo/ManualSyncFragment.kt
Comment (jingtang10) [inline demo/src/main/java/com/google/android/fhir/demo/ManualSyncFragment.kt:1]: rename this file to `SyncFragment.kt`

File: demo/src/main/res/drawable/sync_24dp.xml
Comment (jingtang10) [inline demo/src/main/res/drawable/sync_24dp.xml:1]: this icon looks smaller than the other three on the landing page of the demo app - can you make this a bit bigger (and possibly bolder just to be consistent)?

**Review by jingtang10 — CHANGES_REQUESTED** [review]
If there's nothing else in the activity view model, can we refactor and rename it to sync fragment view model?

Thanks @santosh-pingle!

File: demo/src/main/res/values/strings.xml (hunk 1 above)
Comment (santosh-pingle) [reply demo/src/main/res/values/strings.xml]: done.

File: demo/src/main/java/com/google/android/fhir/demo/ManualSyncFragment.kt
Comment (santosh-pingle) [reply demo/src/main/java/com/google/android/fhir/demo/ManualSyncFragment.kt:1]: done.

File: demo/src/main/res/drawable/sync_24dp.xml
Comment (santosh-pingle) [reply demo/src/main/res/drawable/sync_24dp.xml:1]: done.

**Review by jingtang10 — APPROVED** [review]
Thanks @santosh-pingle

**Pull request merged**